src/Transports/MassTransit.WebJobs.ServiceBusIntegration/Configuration/AzureFunctionsBusConfigurationExtensions.cs

```diff
@@ -51,8 +51,8 @@ public static class AzureFunctionsBusConfigurationExtensions
 
                         if (string.IsNullOrWhiteSpace(connectionString))
                         {
-                            var ns = config["ServiceBusConnection__fullyQualifiedNamespace"]
-                                ?? config[$"{connectionStringConfigurationKey}__fullyQualifiedNamespace"];
+                            var ns = Environment.GetEnvironmentVariable("ServiceBusConnection__fullyQualifiedNamespace")
```

Comment:
config should include Environment variables: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.environmentvariables.environmentvariablesconfigurationprovider?view=dotnet-plat-ext-7.0

I suspect how it is done is incorrect, should be `ServiceBusConnection:` instead of `__`, like:
```csharp
config["ServiceBusConnection:fullyQualifiedNamespace"]
config[$"{connectionStringConfigurationKey}:fullyQualifiedNamespace"]
```

Reply:
Tested and does indeed work this way. Pushed new change